spec/serializers/project_serializer_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ProjectSerializer do
+  let!(:user) { create(:user) }
+  let(:project) { create(:project) }
+  let(:serializer) { ProjectSerializer }
+
+  context 'another way to test' do
+    it 'checks hash data normal attributes' do
```

Comment:
what do you mean by normal ? maybe this is juts fine
```suggestion
    it 'checks hash data attributes' do
```